educhain/engines/qna_engine.py

```
@@ -16,8 +16,9 @@
 from langchain.schema import HumanMessage
 from educhain.core.config import LLMConfig
 from educhain.models.qna_models import (
-    MCQList, ShortAnswerQuestionList, TrueFalseQuestionList, 
-    FillInBlankQuestionList, MCQListMath, Option ,SolvedDoubt, SpeechInstructions
+    MCQList, ShortAnswerQuestionList, TrueFalseQuestionList,
+    FillInBlankQuestionList, MCQListMath, Option ,SolvedDoubt, SpeechInstructions,
```

Ruff (0.8.2)

`educhain.models.qna_models.SpeechInstructions` imported but unused

Remove unused import

(F401)